language/src/regent/std.t

```diff
@@ -3150,38 +3150,6 @@ function std.register_variant(variant)
   variants:insert(variant)
 end
 
-local function make_task_wrapper(task_body)
```

Comment:
I'm fine with changing the public interface to `variant:make_wrapper()` as in a later commit, but personally I'd like to keep this function as it is (just move it to the other file, immediately before the definition of `variant:make_wrapper()`) because it's such a clear, well-defined interface. This is code I can copy-and-paste anywhere and have be useful, while the method version needs to be disentangled from its context.

Reply:
Done